### PR TITLE
Directory Structure

### PR DESCRIPTION
**Pull request by lachfoy**
Renamed branch to tackle sub problems. This PR aims to reorganize the directory structure so that .h files are in a new directory called /include. This directory is then specified in the build command (`-Iinclude`).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,5 +3,5 @@ An application to showcase and execute Splashkit games
 ## Install
 + Install the [SplashKit](https://splashkit.io) SDK using the [guide](https://splashkit.io/articles/installation/)
 ## Run
-+ Compile the application with the command ```skm clang++ program.cpp -o test -lstdc++fs```  
++ Compile the application with the command ```skm clang++ program.cpp -o test -Iinclude -lstdc++fs```
```
Comment (Delcari) [inline README.md:6]: Just update this for good measure to avoid potential case-sensitivity issues. 
```program.cpp -> Program.cpp```

**Review by Delcari — APPROVED** [review]
Haven't looked deeply into the changes on Menu.h, assuming you just relocated the file. 
In future try using the command ```git mv``` [(source)](https://git-scm.com/docs/git-mv) next time to avoid git from interpreting the moved file as a new file.
Otherwise compiles and runs fine on:
Windows ✔️ 
Ubuntu w/ WSL ✔️ 

Happy to merge 😃 

**NOTE**: For some reason the application compiles fine without the additional argument ```-Iinclude``` but I assume in normal circumstances it is required.

File: README.md (hunk 1 above)
Comment (lachfoy) [reply README.md]: Thanks @Delcari 😄 , I actually had no idea about `git mv`. Will go ahead and merge it then if no issues.

**Comment by lachfoy** [comment]
Thanks @Delcari 😄 , I actually had no idea about git mv. Will go ahead and merge it then if no issues.

**Pull request merged**